tools/bumper-bot/bumper.go

```diff
@@ -0,0 +1,104 @@
+//usr/bin/env go run $0 "$@"; exit
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/kubevirt/cluster-network-addons-operator/tools/bumper-bot/parser"
+)
+
+func main() {
+	if len(os.Args) != 2 {
+		fmt.Println("Error: must contain path to components yaml", os.Args)
```

Comment:
It should exit with non-0 return code then, not just return.

Could we use `log` instead? I think it is more suitable to describe the progress of execution. Print is more suitable if you want to return some value to the caller.

Using log would also allow you to call `log.Fatal` here